doc/source/ray-air/examples/torch_image_batch_pretrained.py

```diff
@@ -10,20 +10,20 @@
 from ray.data.preprocessors import BatchMapper
 
 
-def preprocess(batch: np.ndarray) -> pd.DataFrame:
+def preprocess(image_batch: np.ndarray) -> np.ndarray:
     """
-    User Pytorch code to transform user image. Note we still use pandas as
-    intermediate format to hold images as shorthand of python dictionary.
+    User Pytorch code to transform user image with outer dimension of batch size.
     """
     preprocess = transforms.Compose(
         [
-            transforms.ToTensor(),
```

Comment:
don't we have to rescale to range [0,255]? this is what torchvision ToTensor is doing https://pytorch.org/vision/main/generated/torchvision.transforms.ToTensor.html

Reply:
for this example our images are RGB already and falls into [0, 255] range. It's a bit annoying that torchvision's `ToTensor` doesn't work with batch of images with outer dimension while pytorch vision models do ..